### PR TITLE
more components and tests

### PR DESCRIPTION
**Pull request by HBMattoon**
things to be aware of: in this pull request I altered the target element id from 'test' to 'feature' when rendering. The server port has also been update to 3002 so as to no longer conflict with the services.

TODO.
move icons to s3
more tests
some minor tweaks to existing components
implement 'own' dropdown menu
move css for body element over to a styled component

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
-<div id="test"></div>
+<div id="feature"></div>
 <script type="text/javascript" src="./bundle.js"></script>
```
Comment (willresen) [inline client/dist/index.html]: You do not need to define the script type - in fact, it's recommended that you just use `<script src="bundle.js></script>`

File: client/src/app.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      film: {},
+    }
+    this.fetchRandFeaturedFilm = this.fetchRandFeaturedFilm.bind(this);
+  }
+
+  componentDidMount(){
+    console.log('fetch random film');
```
Comment (willresen) [inline client/src/app.jsx]: Be sure to remove unnecessary `console.log`s before production.

File: client/src/component/description.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { DescriptionBox, ToggleMore } from '../style.jsx';
+
+const reducedDesc = function(desc){
```
Comment (willresen) [inline client/src/component/description.jsx]: Can you add some clarifying comments for this function?

File: client/src/component/description.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  let descArr = desc.split('');
+  for (var x = 210; x > 0; x -= 1) {
+    if (descArr[x] === ' ') {
+       descArr = descArr.slice(0,x);
+       x = -1;
+    }
+  }
```
Comment (willresen) [inline client/src/component/description.jsx]: Several missing semi-colons in this file.

File: client/dist/index.html (hunk 1 above)
Comment (HBMattoon) [reply client/dist/index.html]: Ah gotcha, I will fix that right away

File: client/src/component/description.jsx (hunk 3 above)
Comment (HBMattoon) [reply client/src/component/description.jsx]: will do. until then, that function splits the description at the closest empty space before 210 characters, which is then displayed before the user presses 'More'

**Pull request merged**